README.md

```diff
@@ -174,4 +174,9 @@ struct SubCommandTwo {
 }
 ```
 
+It is also possible to retrieve just the names of the arguments and switches 
+passed at runtime, by including the switch `--dump_args_passed` in the args, similarly
```

Comment:
I think it'd be more conventional if this was `--dump-args-passed`, rather than snake_case.

Reply:
Done